src/pages/sectionMeeting/sectionMeeting.js

```diff
@@ -37,8 +48,34 @@ router.post(
   '/',
   asyncErrorHandler(async (req, res, next) => {
     if (req.body.__action === 'newSwipe') {
+
+      var uin = req.body.UIN.trim();
+
+      // attempt to match by netid if it does not start with a number
+      if (uin.length > 0 && Number.isNaN(parseInt(uin[0],10))) {
+        const params = {
+          netid: uin,
+          ciTerm: req.body.ciTerm,
+          ciName: req.body.ciName,
+          ciYear: Number.parseInt(req.body.ciYear, 10),
+        };
+        
+        const results = await dbDriver.asyncQuery(sql.find_matching_student, params);
+        if (results.rows.length > 0) {
```

Comment:
@JParisFerrer we don't handle the case when no matching student is found and `uin` is a netid and not a valid uin, do we?

Reply:
If you don't account for it, you can just use `ERR` to show that an invalid `netId` was entered.

Reply:
https://github.com/nwalters512/attendance/blob/200f535313ff25596cab71bf9024da78df6f05ea/src/pages/home/home.js#L28

Reply:
This should be handled by L85 (and seems to be for me)

Reply:
Could change for code clarity though

Reply:
done